<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ROUTAGE_Editor_ui</class>
 <widget class="QDialog" name="ROUTAGE_Editor_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>501</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Route Parameters</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Name</string>
   </property>
   <widget class="QLineEdit" name="editName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="editBoat">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Points de depart et d'arrivee</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>20</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QComboBox" name="fromPOI"/>
     </item>
     <item>
      <widget class="QComboBox" name="toPOI"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>81</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btOk">
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btCancel">
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>481</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Taille et couleur du resultat du routage</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>60</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Date et heure de depart du routage</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>287</width>
      <height>75</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="1">
      <widget class="QDateTimeEdit" name="editDateBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::UTC</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>481</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Duree Isochrones</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pas Anglulaire</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Angle Balayage</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="duree">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string extracomment="min"> min</string>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="maximum">
     <number>600</number>
    </property>
    <property name="value">
     <number>60</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="range">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string extracomment="deg"> deg</string>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="minimum">
     <number>30</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>120</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="step">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string> deg</string>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>180.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.000000000000000</double>
    </property>
    <property name="value">
     <double>5.000000000000000</double>
    </property>
   </widget>
   <widget class="QCheckBox" name="windForced">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Forcer le vent</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>TWS</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="TWS">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>61</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string extracomment="nds"> nds</string>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>60.000000000000000</double>
    </property>
    <property name="value">
     <double>20.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>TWD</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="TWD">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string extracomment="deg"> deg</string>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Coeff Exploration</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="explo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="showIso">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Montrer les isochrones</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="convRoute">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Convertir en Route</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="useVac">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>261</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Utiliser la duree des vacations entre 2 isochrones</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>editName</tabstop>
  <tabstop>editBoat</tabstop>
  <tabstop>fromPOI</tabstop>
  <tabstop>toPOI</tabstop>
  <tabstop>editDateBox</tabstop>
  <tabstop>duree</tabstop>
  <tabstop>step</tabstop>
  <tabstop>range</tabstop>
  <tabstop>explo</tabstop>
  <tabstop>windForced</tabstop>
  <tabstop>TWS</tabstop>
  <tabstop>TWD</tabstop>
  <tabstop>showIso</tabstop>
  <tabstop>convRoute</tabstop>
  <tabstop>btOk</tabstop>
  <tabstop>btCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btOk</sender>
   <signal>clicked()</signal>
   <receiver>ROUTAGE_Editor_ui</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCancel</sender>
   <signal>clicked()</signal>
   <receiver>ROUTAGE_Editor_ui</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btDeleteClicked()</slot>
 </slots>
</ui>
